apps/loan/src/components/PageCrvUsdStaking/Page.tsx

```diff
@@ -0,0 +1,79 @@
+import type { NextPage } from 'next'
+
+import { t } from '@lingui/macro'
+import { useEffect } from 'react'
+import Image from 'next/image'
+import { useLocation, useNavigate, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { scrollToTop } from '@/utils/helpers'
+import usePageOnMount from '@/hooks/usePageOnMount'
+
+import { RCScrvUSDLogoSM } from 'ui/src/images'
+
+import DocumentHead from '@/layout/DocumentHead'
+import ExternalLink from '@/ui/Link/ExternalLink'
+import Box from '@/ui/Box'
+import Settings from '@/layout/Settings'
+
+import CrvUsdStaking from '@/components/PageCrvUsdStaking'
+
+const Page: NextPage = () => {
+  const params = useParams()
+  const location = useLocation()
+  const navigate = useNavigate()
+  const { routerParams } = usePageOnMount(params, location, navigate)
+
+  useEffect(() => {
```

Comment:
do we need this in every page? It would make sense to have it in the app root ([example](https://stackoverflow.com/a/73289884/977026))

Reply:
Excellent suggestion! But I think it should be done in a different PR